OTKit/aliases.yml

```diff
@@ -37,40 +37,40 @@ aliases:
   # Spacing
   # #############################################
   spacing-xsmall:
-    value: "0.25rem"
+    value: "4px"
   spacing-small:
-    value: "0.5rem"
+    value: "8px"
   spacing-medium:
-    value: "1rem"
+    value: "16px"
   spacing-large:
-    value: "2rem"
+    value: "32px"
   spacing-xlarge:
-    value: "4rem"
+    value: "64px"
 
   # #############################################
   # Typography
   # #############################################
   font-size-xsmall:
-    value: "0.875rem"
+    value: "14px"
   font-size-small:
-    value: "1rem"
+    value: "16px"
   font-size-medium:
-    value: "1.125rem"
+    value: "18px"
   font-size-large:
-    value: "1.5rem"
+    value: "24px"
   font-size-xlarge:
-    value: "3rem"
+    value: "48px"
   # line heights seem arbitrary because they are mapped to specific font sizes
   line-height-xsmall:
-    value: "1.43"
+    value: "16px"
```

Comment:
You are changing the behavior of line heights with these changes? These line height values used to be numerical which means they scale (multiply) with the corresponding font size to get the final line height in px value.

If you do the line height calculation based on your own pixel values changed above, none of them seem to line up. For example, you set `font-size-xsmall: 14px`, then its line height should be `14px * 1.43 = 20.02px`

Reply:
Ah, that is totally my bad.  I did not realize that and replaced them with what is in the design document.

I don't think this is super clear though and am still a little confused.  This doesn't seem to line up with what design has speced out for the line-heights.  How were these ratios determined to begin with?

Reply:
Here's the current design spec:

![image](https://user-images.githubusercontent.com/4041776/31961356-587e41f0-b8af-11e7-8d30-70ab295e0625.png)

Reply:
Regarding the line-heights, talked with Eric a bit, and I think it should be safe for us to simply update this to be in line with what design has speced out for line heights in pixels.  Everyone is still just doing things like `line-height: $lineHeightSmall`.

I'm also not sure how the current ratios were set up, since they don't seem to match up with the spec either way after doing the math.

Reply:
I think it's probably ok as this will be a major version bump anyway, so we can adjust the line heights to match the current spec and verify the pages as we bump up to the major version.

Reply:
You should talk to @francesyun because the spec is outdated and the px line height aint right @cbeley